### DISCUSSION TITLE
Data Doesn't Save To Repo

### DISCUSSION THREAD

**Discussion by flikites** [JamesIves/fetch-api-data-action#632] [Q&A]
The log says the data was saved, but no files were added or updated. 

Do I need to use a particular github secret in order for it to work? 

This is my action file: https://github.com/FliKites/hns-price-updater/blob/main/.github/workflows/go.yaml

**Comment by JamesIves** [comment]
The data is saving to that particular workflow run, once it's terminated it will no longer exist. You need to push that data to a specific branch using a deployment job. You can see an example of this [here](https://github.com/JamesIves/fetch-api-data-action#getting-started-%EF%B8%8F) within the getting started section of the readme (Click on `You can view a full example of this here.`) 

Lemme know if that helps!
